nat46/modules/nat46-netdev.c

```diff
@@ -251,6 +251,47 @@ int nat46_configure(char *devname, char *buf) {
 	}
 }
 
+int nat46_remove(char *devname, char *buf) {
+	int ret = -1;
+	int buflen = 1024;
+	char config_remove[buflen];
+	struct net_device *dev;
+	nat46_instance_t *nat46;
+	nat46_instance_t *nat46_remove;
+	int result_rem;
+	int i;
+
+	if((dev = find_dev(devname)) == NULL ||
+	   (nat46 = netdev_nat46_instance(dev)) == NULL ||
```

Comment:
(1) You can't take a nat46 pointer here without incrementing the reference on it, see below